geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala

```diff
@@ -51,25 +51,32 @@ object IngestCommand {
     @Parameter(names = Array("-is", "--index-schema"), description = "GeoMesa index schema format string")
     var indexSchema: String = null
 
-    @Parameter(names = Array("-cols", "--columns"), description = "the set of column indexes to be ingested, must match the SimpleFeatureType spec")
+    @Parameter(names = Array("-cols", "--columns"), description = "the set of column indexes to be ingested, " +
+      "must match the SimpleFeatureType spec (zero-indexed)")
```

Comment:
The Github word wrap is around 100 characters for me.  The last thing I see is "ingested," on the previous line.